radix-engine-stores/src/rocks_db.rs

```diff
@@ -62,6 +62,22 @@ impl CommittableSubstateDatabase for RocksdbSubstateStore {
     }
 }
 
+impl ListableSubstateDatabase for RocksdbSubstateStore {
+    fn list_partition_keys(&self) -> Box<dyn Iterator<Item = DbPartitionKey> + '_> {
+        let mut partition_keys: Vec<DbPartitionKey> = vec![];
+        // Eliminate duplicated keys (if any)
```

Comment:
The duplicates are kinda inherent, since a single partition is meant to hold potentially unbounded number of substates under different sort keys (by design).

Rocksdb iterates sorted entries (by design), so `.dedup()` would actually be very appropriate 👍

Reply:
Alright. Will change to `dedup()`